src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2215,7 +2382,7 @@ def __init__(self, config: Mask2FormerConfig):
         super().__init__(config)
         self.pixel_level_module = Mask2FormerPixelLevelModule(config)
         self.transformer_module = Mask2FormerTransformerModule(in_features=config.feature_size, config=config)
-
+        self.video_transformer_module = Mask2FormerVideoTransformerModule(in_features=config.feature_size, config=config)
```

Comment:
It'd be preferable to not modules unless they will be used during inference. Could you convert this to an if-else statement instead?

```
        if config.video_input:
             self.transformer_module = Mask2FormerTransformerModule(in_features=config.feature_size, config=config)
        else:
             self.transformer_module = Mask2FormerVideoTransformerModule(in_features=config.feature_size, config=config)
```

Reply:
aah yes, I get my mistake. I'll fix this.